indico/modules/events/blueprint.py

```diff
@@ -60,7 +60,21 @@
 # Misc
 _bp.add_url_rule('/event/<int:event_id>/key-access', 'key_access', RHEventAccessKey, methods=('POST',))
 _bp.add_url_rule('/event/<int:event_id>/api/info-for-series', 'single_event_api', RHSingleEventAPI)
-_bp.add_url_rule('/event/<int:event_id>/check-email', 'check_email', RHEventCheckEmail)
+
+# Email checker for PersonLinkField
+# Depending on where the person link is used, the person link field generates the correct url.
+# Eeach object has its own url so that we can do proper access checks.
+event_object_url_prefixes = {
```

Comment:
please underscore this; it's pretty specific to this one and unlike the other one shouldn't be accidentally imported